content/clause-interleaving.md

```diff
@@ -1,8 +1,9 @@
 ---
 layout: sip
-title: SIP-NN Clause Interleaving
-stage: design
-permalink: /sips/:title.html
+title: SIP-47 Clause Interleaving
```

Comment:
I'd like to mention once again that it should be "Parameter Clause Interleaving" as we're not doing that with if-clauses here.

Reply:
I appreciate your concern, however this has not caused confusion for anyone else, and there is precedent for omitting "parameter" from the name:
https://docs.scala-lang.org/scala3/reference/contextual/using-clauses.html
I will therefore not change the name for now, but I encourage affected others to comment on this potential problem

Reply:
> I appreciate your concern, however this has not caused confusion for anyone else, and there is precedent for omitting "parameter" from the name: https://docs.scala-lang.org/scala3/reference/contextual/using-clauses.html I will therefore not change the name for now, but I encourage affected others to comment on this potential problem

it's not about omitting the word "parameter", it's about naming the type of clause. the word "using" names the clause in your other example.